### PR TITLE
Marketplace move hashVersion

### PR DESCRIPTION
**Pull request by NicolasMahe**
Dependant on https://github.com/mesg-foundation/core/pull/864.
Closes https://github.com/mesg-foundation/core/issues/861.

This PR move the `hashVersion` from the manifest data to the service definition. Like this, the core itself can manage the version used to calculate the service's hash.

### CODE REVIEW

**Comment by NicolasMahe** [comment]
@mesg-foundation/core ready to review

**Comment by NicolasMahe** [comment]
@antho1404 do you agree with this modification?

**Comment by antho1404** [comment]
let's rename `hashVersion` into `version` this is a version of a service not only the hash

File: service/service.go [hunk 1]
```diff
@@ -27,6 +27,9 @@ type Service struct {
 	// It represents the service uniquely.
 	Hash string `hash:"-"`
 
+	// HashVersion is the version of the algorithm used to calculate the of the hash
+	HashVersion string `hash:"-"`
```
Comment (antho1404) [inline service/service.go]: I don't think `hash:"-"` is used anymore but just to keep in consistent, let's pretend it is and in that case the version is required in the hashing

**Comment by NicolasMahe** [comment]
Closing in favor of https://github.com/mesg-foundation/core/pull/906

**Pull request closed without merging**